pkg/virt-api/webhooks/hyperv_test.go

```diff
@@ -0,0 +1,265 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2019 Red Hat, Inc.
+ *
+ */
+
+package webhooks
+
+import (
+	"fmt"
+	"reflect"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	k8sfield "k8s.io/apimachinery/pkg/util/validation/field"
+
+	v1 "kubevirt.io/client-go/api/v1"
+)
+
+var _false bool = false
+
+var _ = Describe("Mutating Webhook HyperV utils", func() {
+
+	It("Should not mutate VMIs without HyperV configuration", func() {
```

Comment:
I think these functional tests belong in the respective vmi validation and mutation unit test suites that already exist. The reason being, we want to test mutation and validation of hyperv in the context of the full mutation and validation flows for a vmi.

Reply:
ack, moving them